### PR TITLE
Add Git to mandatory dependencies.

### PR DESCRIPTION
**Pull request by DimasInchidi**
# Closes: #3184
# What's Changed
Installation instruction in documentation

# TODO
This might not the fix wanted for #3184 but it's necessary for new developer as long as the error exists. thus, further discussion might be needed.

### CODE REVIEW

**Comment by bryanculver** [comment]
Thanks for the PR @DimasInchidi! Mind adding a changelog fragment for #3184: https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments

**Review by bryanculver — CHANGES_REQUESTED** [review]
Missing changelog fragment

File: nautobot/docs/installation/index.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The following minimum versions are required for Nautobot to operate:
 | PostgreSQL | Database     | 9.6             |
 | MySQL      | Database     | 8.0             |
 | Redis      | Cache, Queue | 4.0             |
+| Git        | Additional   | 2.19            |
```
Comment (bryanculver) [inline nautobot/docs/installation/index.md:21]:
```suggestion
| Git        | Additional   | 2.0            |
```

Any version 2.0+ should suffice.

File: nautobot/docs/installation/index.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ The following minimum versions are required for Nautobot to operate:
 | PostgreSQL | Database     | 9.6             |
 | MySQL      | Database     | 8.0             |
 | Redis      | Cache, Queue | 4.0             |
+| Git        | Additional   | 2.0            |
```
Comment (gsnider2195) [inline nautobot/docs/installation/index.md:21]:
```suggestion
| Git        | Additional   | 2.0             |
```

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**